### PR TITLE
fix(platform): add aria-description for select all checkbox

### LINKED ISSUE #9997: [Reopen][Sourcing][Accessibility][High][Jul] Chooser hierarchy ATL issue 4 - checkbox states
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
ATL testing issue 4

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 15
core./ platform: 0.39.0

#### If this is a bug, please provide steps for reproducing it.
4) On accessing the ‘Select all check box’ on the header column, its label provided ‘Check box’ is incorrect as users will not know the difference between cell check box and header check box.
image

### PR DESCRIPTION
**Pull request by mikerodonnell89**
fixes #9997 

adds an aria description for the select all checkbox on platform table

### CODE REVIEW

**Review by InnaAtanasova — APPROVED** [review]

**Comment by N1XUS** [comment]
@mikerodonnell89 fd-checkbox does not receive focus if keyboard navigation is used. `th` itself should also have label or labelledby attribute

File: libs/i18n/src/lib/languages/georgian.ts [hunk 1]
```diff
@@ -457,7 +457,9 @@ export const FD_LANGUAGE_GEORGIAN: FdLanguage = {
         sortDialogSortByHeader: 'დალაგებულია',
         sortDialogNotSortedLabel: '(დაულაგებელია)',
         sortDialogConfirmBtnLabel: 'კარგი',
-        sortDialogCancelBtnLabel: 'გაუქმება'
+        sortDialogCancelBtnLabel: 'გაუქმება',
+        selectAllCheckboxLabel: 'Select all',
+        deselectAllCheckboxLabel: 'Deselect all'
```
Comment (g-cheishvili) [inline libs/i18n/src/lib/languages/georgian.ts:462]:
```suggestion
        selectAllCheckboxLabel: 'ყველას არჩევა',
        deselectAllCheckboxLabel: 'ყველას გადარჩევა'
```

File: libs/i18n/src/lib/languages/russian.ts [hunk 2]
```diff
@@ -610,7 +610,9 @@ export const FD_LANGUAGE_RUSSIAN: FdLanguage = {
         sortDialogSortByHeader: 'Сортировать по',
         sortDialogNotSortedLabel: '(Не отсортировано)',
         sortDialogConfirmBtnLabel: 'ОК',
-        sortDialogCancelBtnLabel: 'Отменить'
+        sortDialogCancelBtnLabel: 'Отменить',
+        selectAllCheckboxLabel: 'Select all',
+        deselectAllCheckboxLabel: 'Deselect all'
```
Comment (g-cheishvili) [inline libs/i18n/src/lib/languages/russian.ts:615]:
```suggestion
        selectAllCheckboxLabel: 'Выбрать все,
        deselectAllCheckboxLabel: 'Отменить все'
```

File: libs/i18n/src/lib/languages/ukrainian.ts [hunk 3]
```diff
@@ -608,7 +608,9 @@ export const FD_LANGUAGE_UKRAINIAN: FdLanguage = {
         sortDialogSortByHeader: 'Сортувати за',
         sortDialogNotSortedLabel: '(Не відсортовано)',
         sortDialogConfirmBtnLabel: 'OK',
-        sortDialogCancelBtnLabel: 'Скасувати'
+        sortDialogCancelBtnLabel: 'Скасувати',
+        selectAllCheckboxLabel: 'Select all',
+        deselectAllCheckboxLabel: 'Deselect all'
```
Comment (g-cheishvili) [inline libs/i18n/src/lib/languages/ukrainian.ts:613]:
```suggestion
        selectAllCheckboxLabel: 'Вибрати все',
        deselectAllCheckboxLabel: 'Зняти вибір із усіх'
```

Although, deselect all sounds strange, it literally means "remove selection from all", @N1XUS HELP

**Review by g-cheishvili — APPROVED** [review]
I only speak Russian and Georgian from the listed languages and translated on those languages. Code related to the task itself is acceptable

**Pull request merged**